client/my-sites/domains/domain-management/edit/domain-types/transfer-in-domain-type.jsx

```diff
@@ -0,0 +1,231 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import VerticalNav from 'components/vertical-nav';
+import VerticalNavItem from 'components/vertical-nav/item';
+import { withLocalizedMoment } from 'components/localized-moment';
+import DomainStatus from '../card/domain-status';
+import QuerySitePurchases from 'components/data/query-site-purchases';
+import {
+	getByPurchaseId,
+	isFetchingSitePurchases,
+	hasLoadedSitePurchasesFromServer,
+} from 'state/purchases/selectors';
+import { transferStatus } from 'lib/domains/constants';
+import page from 'page';
+import { domainManagementTransferInPrecheck } from 'my-sites/domains/paths';
+import { isCancelable } from 'lib/purchases';
+import { cancelPurchase } from 'me/purchases/paths';
+import RemovePurchase from 'me/purchases/remove-purchase';
+import { INCOMING_DOMAIN_TRANSFER_STATUSES } from 'lib/url/support';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class TransferInDomainType extends React.Component {
+	getVerticalNavigation() {
+		return <VerticalNav>{ this.removeTransferNavItem() }</VerticalNav>;
+	}
+
+	removeTransferNavItem() {
+		const { domain, isLoadingPurchase, purchase, selectedSite, translate } = this.props;
+
+		if ( ! domain.currentUserCanManage ) {
+			return null;
+		}
+
+		if (
+			domain.transferStatus === transferStatus.PENDING_OWNER ||
+			domain.transferStatus === transferStatus.PENDING_REGISTRY
+		) {
+			return null;
+		}
+
+		let title;
+
+		if ( domain.transferStatus === transferStatus.CANCELLED ) {
+			title = translate( 'Remove transfer from your account' );
+		} else {
+			title = translate( 'Cancel transfer' );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **26** times:
`translate( 'Cancel Transfer' )` *ES Score: 12*